tests/security/cc/ip_eb_tables.pm

```diff
@@ -23,9 +23,34 @@ sub run {
 
     select_console 'root-console';
 
+    #Configure bridge for ip_eb_tables workaround
+    assert_script_run("echo \"BOOTPROTO='dhcp'\" > /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"STARTMODE='auto'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE='yes'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE_PORTS='eth0'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE_STP='off'\" >> /etc/sysconfig/network/ifcfg-br0");
```

Comment:
Hi Viktor, 

I suggest you can use one command line to modify the file like below code:

"os-autoinst-distri-opensuse/tests /console/rsync_server.pm"

    script_run 'cat > /etc/rsyncd.conf <<EOF
max connections = 2
log file = /var/log/rsync.log
timeout = 300

[pub]
     comment = Testing files available for download
     read only = yes
     list = yes
     path = /srv/rsync_test/pub
     uid = nobody
     gid = nobody
     auth users = test42
     secrets file = /etc/rsyncd.secrets

EOF
true';

Reply:
Fixed